tests/unit/test_aggregation.py

```diff
@@ -292,12 +307,17 @@ def test_iterator_constructor_defaults():
     assert iterator._more_results
     assert iterator._retry is None
     assert iterator._timeout is None
+    assert iterator._explain_metrics is None
 
 
 def test_iterator_constructor_explicit():
     query = object()
     client = object()
-    aggregation_query = AggregationQuery(client=client, query=query)
+    explain_options = object()
+    aggregation_query = AggregationQuery(
```

Comment:
just to confirm, passing through explain_options is not now required, right? This is just testing all values in the construtor?

Reply:
yeah explain_options is an optional argument, so it defaults to `None` if not passed. This is just checking to make sure if it is passed, it is used